canister/src/state.rs

```diff
@@ -89,6 +93,22 @@ impl State {
     pub fn get_utxos(&self, address: Address) -> AddressUtxoSet<'_> {
         AddressUtxoSet::new(address, &self.utxos, &self.unstable_blocks)
     }
+
+    /// Return 'true' if the difference between the maximum height of all block
+    /// headers and the maximum height of all unstable blocks is at most g.
```

Comment:
`g` is not defined here.

```suggestion
    /// Returns 'true' if the difference between the maximum height of all block
    /// headers and the maximum height of all unstable blocks is at most the syncing threshold.
```